README.md

```diff
@@ -16,16 +16,11 @@ This plugin improves the Mezzurite SDK implementation experience.  The Mezzurite
 
  Updates in Component Tracking Status on the fly whenever a component file or its external html template is changed, deleted or a new component is created.
 
- ##### Demo
- ![Components Demo](https://github.com/Microsoft/vscode-mezzurite/blob/master/resources/components_demo.gif?raw=true)
  
  ####  Modules Tracking
  Displaying the modules configured with mezzurite performance markers to capture ALT and VLT for the application code currently active in editor workspace.
 
- Updates in Modules Tracking Status on the fly whenever a module file is changed, deleted or a new module is created. Hovering over the module name displays the missing condition to track that module.
-
- ##### Demo
- ![Modules Demo](https://github.com/Microsoft/vscode-mezzurite/blob/master/resources/modules_demo.gif?raw=true)
+ Updates in Modules Tracking Status on the fly whenever a module file is changed, deleted or a new module is created. 
```

Comment:
Can you rework this sentence as well as line 17? There isn't a "Module Tracking Status" section anymore, and it's rather hard to understand.